File: python/exercício_48.py
n1=int(input("Digite o numero que deseja saber se é primo: "))
tot = 0
for c in range(1, n1 + 1):
            if n1 % c == 0:
                tot += 1
if tot == 2:
    print("O numero {} é primo".format(n1))
else:
    print("O numero {} não é primo".format(n1))

File: java/exercício_48.java
import java.util.Scanner;

public class exercício_48 {
    public static void main(String[] args) {

        try (Scanner numero = new Scanner(System.in)){

        System.out.printf("Digite o numero que deseja saber se é primo: ");
        int primo = numero.nextInt();
        int total = 0;
        for (int c = 1; c <= primo; c++) {
            if (primo % c == 0) {
                total++;
            }
        }
        if (total == 2) {
            System.out.printf("O numero: " + primo + " É um numero primo");
        } else {
            System.out.printf("Não é um numero primo!");
        }

    }
}
}